modules/serverless-describe-subs-kn.adoc

```diff
@@ -6,11 +6,11 @@
 [id="serverless-describe-subs-kn_{context}"]
 = Describing subscriptions by using the Knative CLI
 
-You can print information about a subscription in the terminal by using the `kn` CLI.
+Using the `kn` CLI to describe subscriptions provides a more streamlined and intuitive user interface over viewing YAML files directly. You can use the `kn subscription describe` command to print information about a subscription in the terminal by using the `kn` CLI.
```

Comment:
Same here with _files uses_ and second sentence.

Reply:
See other similar comment response